app/src/main/java/org/jellyfin/androidtv/util/apiclient/ApiClientExtensions.kt

```diff
@@ -110,3 +112,26 @@ suspend fun ApiClient.getLocalTrailers(item: BaseItem): List<LocalTrailer>? = su
 		}
 	})
 }
+
+suspend fun ApiClient.getEpisodesOfSeason(episode: Episode): List<Episode>? = if (episode.seasonId != null) getEpisodesOfSeason(episode.seasonId) else null
+
+private suspend fun ApiClient.getEpisodesOfSeason(seasonId: String): List<Episode>? = suspendCoroutine {
+	continuation ->
+	val query = StdItemQuery()
+	query.parentId = seasonId
+	query.includeItemTypes = arrayOf("Episode")
+	query.startIndex = 0
+	query.fields = FIELDS_REQUIRED_FOR_LIFT
+	query.userId = this.currentUserId
+
+	this.GetItemsAsync(query, object : Response<ItemsResult>() {
+		override fun onResponse(response: ItemsResult?) {
+			continuation.resume(response?.items?.map {it.liftToNewFormat() as Episode }?.toList())
+		}
+
+		override fun onError(exception: java.lang.Exception?) {
```

Comment:
```suggestion
		override fun onError(exception: Exception?) {
```

Maybe even

```suggestion
		override fun onError(exception: Exception) {
```

Reply:
There can be null as exception, I'll just drop the java.lang